app-modules/ai/src/Jobs/Advisors/RetryAdvisorMessage.php

```diff
@@ -104,14 +104,13 @@ public function handle(): void
                 hasImageGeneration: $this->hasImageGeneration,
             );
         } catch (Throwable $exception) {
-            report($exception);
+            if ($this->hasImageGeneration && ($this->attempts() < 3)) {
+                $this->release();
```

Comment:
Just checking, is there anything specific about the error that happens that we have seen so we can target it?

Or is it just a generic exception?